src/queries/flatRateFulfillmentMethods.js

```diff
@@ -15,7 +15,28 @@ export default async function flatRateFulfillmentMethods(context, input) {
 
   await context.validatePermissions("reaction:legacy:shippingMethods", "read", { shopId });
 
-  return Shipping.find({
-    shopId
-  });
+  // aggregate pipeline to extract fulfillment methods inside shippment
+  return {
+    collection: Shipping,
+    pipeline: [
+      {
+        $match: {
+          shopId
```

Comment:
Need to add addition match condition here:

```js
"provider.name": "flatRates"
```

Historically, there were other plugins storing their methods in the same collection.